docs/getting-started-guides/rkt/notes.md

```diff
@@ -59,7 +59,7 @@ Under rktnetes, `kubectl get logs` currently cannot get logs from applications t
 
 ## Init containers
 
-The alpha [init container](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/container-init.md) feature is currently not supported.
+The beta [init container](/docs/user-guide/pods/init-containers.md) feature is currently not supported.
```

Comment:
It seems to redirect anyway, but I think the canonical URL should exclude the `.md`: `/docs/user-guide/pods/init-containers/`.